spec/classes/init_spec.rb

```diff
@@ -4,7 +4,15 @@
   context 'supported operating systems' do
     on_supported_os.each do |os, facts|
       context "on #{os}" do
-        let(:facts) { facts }
+        let(:facts) do
+          facts.merge({
+            systemd_internal_services: {
+              "systemd-resolved.service": "enabled",
+              "systemd-networkd.service": "enabled",
+              "systemd-timesyncd.service": "enabled",
```

Comment:
@bastelfreak I don't understand how the tests ever passed (since October 2017) without these!